bundles/EcommerceFrameworkBundle/PriceSystem/ModificatedPriceInterface.php

```diff
@@ -26,6 +28,11 @@ interface ModificatedPriceInterface extends PriceInterface
      * @return string
      */
     public function getDescription();
+
+    /**
+     * @return RuleInterface|null
+     */
+    public function getRule();
```

Comment:
Again, I'd revert this change for this PR.

Reply:
https://github.com/pimcore/pimcore/pull/7549